src/dnf-plugins/subscription-manager.py

```diff
@@ -54,9 +54,9 @@ def __init__(self, base, cli):
         super(SubscriptionManager, self).__init__(base, cli)
         self.base = base
         self.cli = cli
-        self.config()
+        self._config()
```

Comment:
Maybe we should just not call this method on __init__.

Reply:
I already mentioned this in this comment: https://github.com/candlepin/subscription-manager/pull/2000#issuecomment-448604852

See also this: https://github.com/candlepin/subscription-manager/pull/1884 and this: https://bugzilla.redhat.com/show_bug.cgi?id=1609052